lib/gherkin/i18n.json

```diff
@@ -371,7 +371,7 @@
 	  "scenario": "Scenarij",
 	  "scenario_outline": "Skica|Koncept",
 	  "examples": "Primjeri|Scenariji",
-	  "given": "*|Zadan|Zadani|Zadano",
+	  "given": "*|Za dan|Za dani|Za dano",
```

Comment:
removing the previous translations might break existing feature files written using them (at least in the PHP implementation in Behat. I don't how how Cucumber itself deals with it). It may be worth keeping the existing alternatives for BC.
I don't know what the policy of the Cucumber team is for that